specs/interop/token-bridging.md

```diff
@@ -72,26 +101,200 @@ contract OptimismSuperchainERC20 is ERC20, ISuperchainERC20 {
 }
 ```
 
-## L1 Native Tokens
+[Disclaimer]
+
+- Threat the following code as experimental. It will have errors and will change function names. The goal is to discuss functionalities.
+- Current function names are open to discussion.
+
+## Implementation
+
+### Cross Chain transfers
+
+The standard will include the following three external functions:
```

Comment:
I think it will be easier to maintain this spec if we get rid of the solidity code block above that defines the interface and then just define the interface like you did below, but include possible return arguments like:

```
bridgeERC20(uint256 amount, uint256 chainId)(bool)
```